std_msgs/Header header
map_generation/WsSphere[] WsSpheres
float32 resolution
